src/Servers/IIS/build/Build.Common.Settings

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
```

Comment:
Why not import src/Servers/IIS/build/Config.Definitions.Props here❔

Reply:
Great idea, I didn't even notice that existed, will add the arm64 to that and just import that, thanks for pointing that out

Reply:
So looks like the only thing missing from Config.Definitions.Props is the Any CPU configuration, is this something we need or can I just drop Any CPU from all of our projects as part of this cleanup, this isn't something we ship anyways right?

Reply:
We're talking about what VS expects here and I'm not sure what will happen w/o AnyCPU in the options.